module/VuFindTheme/src/VuFindTheme/View/Helper/HeadScript.php

```diff
@@ -103,6 +113,7 @@ public function itemToString($item, $indent, $escapeStart, $escapeEnd)
             }
         }
 
+        $item->attributes['nonce'] = $this->cspNonce;
```

Comment:
I believe that you could also use `$this->getView()->plugin('cspnonce')` here if you want to avoid the extra injection work... though if you prefer to be more explicit, that's fine with me. Just wanted to point out the more concise alternative in case you found it more appealing.

Reply:
I like the explicit injection here... there is possibility that we can avoid using cspnonce view helper even in templates (= inject all sources through other helpers) and than we can even remove nonce helper.

Reply:
Sounds good to me!